File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/MailBlocks.scala
package io.swagger.server.model


/**
 * = MailBlocks =
 *
 * The listing of blocked emails.
 *
 * @param local 
 * @param mbtrap 
 * @param subject 
 */
case class MailBlocks (
  local: List[MailBlockClickHouse],
  mbtrap: List[MailBlockClickHouse],
  subject: List[MailBlockRspamd]
)

File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/MailBlocks.js
goog.provide('API.Client.MailBlocks');

/**
 * The listing of blocked emails.
 * @record
 */
API.Client.MailBlocks = function() {}

/**
 * @type {!Array<!API.Client.MailBlockClickHouse>}
 * @export
 */
API.Client.MailBlocks.prototype.local;

/**
 * @type {!Array<!API.Client.MailBlockClickHouse>}
 * @export
 */
API.Client.MailBlocks.prototype.mbtrap;

/**
 * @type {!Array<!API.Client.MailBlockRspamd>}
 * @export
 */
API.Client.MailBlocks.prototype.subject;

